src/test/java/org/salesforce/runner/RunTests.java

```diff
@@ -1,18 +1,118 @@
 package org.salesforce.runner;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import core.api.ApiManager;
+import core.api.ApiMethod;
+import core.api.ApiRequest;
+import core.api.ApiResponse;
+import core.utils.PropertiesReader;
 import io.cucumber.testng.AbstractTestNGCucumberTests;
 import io.cucumber.testng.CucumberOptions;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpStatus;
+import org.apache.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.DataProvider;
+import salesforce.api.ApiEndPoints;
+import salesforce.api.Authentication;
+import salesforce.api.entities.Account;
+import salesforce.api.entities.Contact;
+import salesforce.api.entities.Response;
+
+import static salesforce.api.Authentication.token;
+import static io.restassured.RestAssured.authentication;
+
+import java.util.Properties;
 
 @CucumberOptions(
         features = {"src/test/resources/features"},
         plugin = {"html:target/site/cucumber-pretty.html", "json:target/cucumber.json"},
         glue = {"org.salesforce"}
 )
 public class RunTests extends AbstractTestNGCucumberTests {
+    public Logger LOGGER = Logger.getLogger(getClass());
+    public ApiRequest apiRequest;
+    public ApiResponse apiResponse;
+    public Authentication authentication = new Authentication();
+    public Account account;
+    public Contact contact;
     @Override
     @DataProvider(parallel = true)
     public Object[][] scenarios(){
         return super.scenarios();
     }
+    @BeforeTest
+    public void loginAndSetup() throws JsonProcessingException {
+        apiRequest = new ApiRequest();
+        Properties properties = PropertiesReader.getProperties("config.properties");
+        authentication.getAuth();
+        apiRequest.addHeader("Content-Type", "application/json")
+                .addHeader(HttpHeaders.AUTHORIZATION, token.getTokenType() + " " + token.getAccessToken())
+                .setBaseUri(token.getInstanceUrl() + properties.getProperty("SERVICE")
+                        + properties.getProperty("VERSION"));
+        createAccount();
+        createContact();
+    }
+
+    @AfterTest
+    public void afterExecution() {
+        deleteAccount();
+        deleteContact();
+        apiRequest = new ApiRequest();
+    }
+
+    public void deleteAccount() {
```

Comment:
Maybe you can create a class with all of this methods, and use that class to create and delete the pre condition objects instead of having all the methods insinde the runner

Reply:
ok Well I implemented anotation and I deleted the lines 57, 58, 63 and 64

Reply:
I think is better